### DISCUSSION TITLE
Dudas sobre parcial

### DISCUSSION THREAD

**Discussion by NicolasPabloW** [dds-jv/foro#168] [Consultas generales] [answered]
¿ El parcial va a tener parte teorica ? 
Eso lo pregunto porque no tendria mucho sentido que lo tuviera si podemos leer los apuntes en el parcial (cosa que se permite segun los lineamientos publicados)

No funciona la webcam de mi laptop.Entraria al meet con desde el celular , tengo el zoom ahi.Pero no entiendo lo de entrar al meet dos veces ¿Eso significa que debo entrar al meet desde el celular y la laptop a la vez? ¿O que quiere decir?

Al estar usando la camara del celular en vez de la webcam de la laptop , yo tengo pensado poner el celular acostado sobre el teclado de esta(sin tocarlo , ) y apoyado sobre la pantalla de esta ,de modo que la camara queda en el lugar de la webcam pero mas abajo (taparia muy poco de mi pantalla , eso no seria problema) ¿Hay que hacerlo asi o de otra forma ?(los lineamientos no lo aclaran) 

¿El enunciado del parcial se entrega en PDF por mail o como?

**Comment by NicolasPabloW** [comment]
La segunda pregunta la acabo de verificar en una reunion de prueba y ya lo entendi.
Las demas dudas se mantienen.

**Comment by gastonprieto** [comment, marked as answer]
> ¿ El parcial va a tener parte teorica ?
Eso lo pregunto porque no tendria mucho sentido que lo tuviera si podemos leer los apuntes en el parcial (cosa que se permite segun los lineamientos publicados)

Si teoría te referís a dar definiciones es un no. Que hayan entendido la teoría lo evaluamos si hacen la práctica bien ya que se basa en los conceptos teoricos que vimos.

> No funciona la webcam de mi laptop.Entraria al meet con desde el celular , tengo el zoom ahi.Pero no entiendo lo de entrar al meet dos veces ¿Eso significa que debo entrar al meet desde el celular y la laptop a la vez? ¿O que quiere decir?

No es obligatorio entrar dos veces, eso es por si queres entrar a la meet con la compu por que te es mas fácil escribir `tengo una duda`. Pero si queres entrar una vez no hay drama.

> Al estar usando la camara del celular en vez de la webcam de la laptop , yo tengo pensado poner el celular acostado sobre el teclado de esta(sin tocarlo , ) y apoyado sobre la pantalla de esta ,de modo que la camara queda en el lugar de la webcam pero mas abajo (taparia muy poco de mi pantalla , eso no seria problema) ¿Hay que hacerlo asi o de otra forma ?(los lineamientos no lo aclaran)

Con que te veas un poco vos estamos bien.

> ¿El enunciado del parcial se entrega en PDF por mail o como?

Esto lo dice los lineamientos, se entregan en el google doc que les vamos a compartir, como lo compartimos nosotros tenemos acceso, y una vez que digas que entregaste el parcial, sacamos el acceso. A ese google doc deberías subir todo, prosa/codigo, imagenes.
